com.krioltech.fnmap.yml

```diff
@@ -0,0 +1,40 @@
+app-id: com.krioltech.fnmap
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: fnmap
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=xdg-documents
+rename-desktop-file: fnmap.desktop
+
+modules:
+  - buildsystem: autotools
+    name: nmap
+    sources:
+      - type: archive
+        url: https://nmap.org/dist/nmap-7.94.tgz
+        sha256: 7ebc3a0d369d5965ba8b6513bce3ff6d3307a7cc87dd18f70d0af5d8e66a2849
+  - buildsystem: simple
+    name: fnmap
+    build-commands:
+      - cp -R fnmap-archive/bin/bundle /app/fnmap
+      - chmod +x /app/fnmap/fnmap
+      - mkdir -p /app/bin/
+      - ln -s /app/fnmap/fnmap /app/bin/fnmap
+      - install -Dm644 fnmap-archive/com.krioltech.fnmap.metainfo.xml -t /app/share/metainfo/
+      - install -Dm644 fnmap-archive/fnmap.desktop /app/share/applications/fnmap.desktop
+      - install -Dm644 fnmap-archive/icons/fnmap.png /app/share/icons/com.krioltech.fnmap.png
```

Comment:
What size is this? This is the wrong location https://docs.flatpak.org/en/latest/conventions.html#application-icons

Reply:
Icon is now installed in the correct location.